log_drains.go

```diff
@@ -15,6 +17,15 @@ type LogDrain struct {
 	URL   string `json:"url"`
 }
 
+type errorRes struct {
+	Error string `json:"error"`
+}
+
+type LogDrainRes struct {
+	LogDrain LogDrain `json:"drain"`
+	Errors   errorRes `json:"errors"`
```

Comment:
But this construction bugs me. Is it really what is returned by the API? If I understand correctly, it means the API may return:

```json
{"errors": {"error": "error message"}}
```

am I right? :thinking: